server/routes/api.php

```diff
@@ -13,6 +13,24 @@
 |
 */
 
-Route::middleware('auth:api')->get('/user', function (Request $request) {
-    return $request->user();
+
+//login
+
+Route::post('/auth/login',  'AuthController@login')->name('login');
+
+//Registro
+
+Route::post('/usuarios/crear', 'UsersController@store')->name('RegistrarUsuario');
+
+//Auth Group
+Route::group(['prefix' => 'auth', 'middleware' => ['jwt.auth', ], ], function () {
+    Route::get('usuario', 'AuthController@usuario')->name('usuarioLogeado');
+    Route::post('renovar', 'AuthController@renovar')->name('renovar');
+    Route::post('logout', 'AuthController@logout')->name('logout');
+});
+
+Route::middleware('jwt.auth')->group(function () {
+    //usuarios Group
+    Route::apiResource('/usuarios', 'UsersController')
+        ->parameters(['usuario' => 'User']);
```

Comment:
El nombre del parámetro de origen (`usuarios`) es igual que el nombre de la ruta (`/usuarios`). El alias debería ser escrito en minúsculas. 

> Cuando se modifique este archivo también debería modificarse el archivo  `UsersController.php`, cambiando los parámetros `$User` y `$usuario` por `$user` para mantener la consistencia.

```suggestion
        ->parameters(['usuarios' => 'user']);
```

Reply:
Solucionado en 663f0e8e0c7d20d72f33c5c9817c172b54659dd0 ✔